common/src/main/resources/db/changelog/v001/AB2D-291-create-initial-tables.sql

```diff
@@ -0,0 +1,197 @@
+--liquibase formatted sql
+--  -------------------------------------------------------------------------------------------------------------------
+
+
+--changeset spathiyil:AB2D-291-CreateSequence-hibernate_sequence failOnError:true dbms:postgresql
+CREATE SEQUENCE hibernate_sequence START WITH 1 INCREMENT BY 1;
+
+
+--changeset spathiyil:AB2D-291-CreateTable-beneficiary failOnError:true dbms:postgresql
+CREATE TABLE beneficiary
+(
+    id                  BIGSERIAL,
```

Comment:
@spathiyil I think this needs to become `bigint` since we are using a sequence, and not relying on auto-increment.

Reply:
done